lib/diagnostic.rb

```diff
@@ -67,8 +84,8 @@ def initialize
 
 ##
 # your answers here
-Answer.housecat_noise = nil
-#
+Answer.housecat_noise = "I am a #{self.class.name}, and I go '#{@sound}'"
+# Because HouseCat inherits from Cat which inherits from Animal which has say_hello and responds with the above answer.
```

Comment:
Also somewhat close. `Answer.housecat_noise = "I am a HouseCat, and I go 'meow'"`